src/uipath/_cli/_runtime/_runtime.py

```diff
@@ -76,31 +76,26 @@ async def execute(self) -> Optional[UiPathRuntimeResult]:
 console = ConsoleLogger()
 
 
-def get_user_script(directory: str, entrypoint: Optional[str] = None) -> Optional[str]:
-    """Find the Python script to process."""
+def get_user_scripts(directory: str, entrypoint: Optional[str] = None) -> List[str]:
+    """Find all Python scripts to process."""
     if entrypoint:
         script_path = os.path.join(directory, entrypoint)
         if not os.path.isfile(script_path):
             console.error(
                 f"The {entrypoint} file does not exist in the current directory."
             )
-            return None
-        return script_path
+            return []
+        return [script_path]
 
     python_files = [f for f in os.listdir(directory) if f.endswith(".py")]
 
     if not python_files:
         console.error(
             "No python files found in the current directory.\nPlease specify the entrypoint: `uipath init <entrypoint_path>`"
         )
-        return None
-    elif len(python_files) == 1:
-        return os.path.join(directory, python_files[0])
-    else:
-        console.error(
-            "Multiple python files found in the current directory.\nPlease specify the entrypoint: `uipath init <entrypoint_path>`"
-        )
-        return None
+        return []
+
+    return [os.path.join(directory, f) for f in python_files]
```

Comment:
Beautiful! :)

However, I do think we need to filter this. The ScriptExecutor does this:
```
        for func_name in ["main", "run", "execute"]:
            if hasattr(module, func_name):
```

so let's ensure that the files have these defined, yea?